.circleci/config.yml

```diff
@@ -15,6 +15,13 @@ jobs:
           name: install
           command: |
             make install install-development
+      - run:
+          name: Install Chrome
+          command: |
+            curl -L -o google-chrome.deb https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
+            sudo dpkg -i google-chrome.deb
+            sudo sed -i 's|HERE/chrome\"|HERE/chrome\" --disable-setuid-sandbox|g' /opt/google/chrome/google-chrome
+            rm google-chrome.deb
```

Comment:
Yes, the version of chrome and chromedriver have to match exactly, and they were breaking CI for a bit when they were out of sync. Things might be updated by now, though.

Reply:
I’m pretty sure CircleCI has support for custom `Dockerfile`s. That might help us specify an exact version. Or perhaps we can specify a URL for the exact version here? If I’m understanding correctly, CI will stop working as soon as Google releases a new version of Chrome, right?

Reply:
Yeah, I believe that would break it. I tried to find a way to get a URL for an exact version but it didn't look like Google supplies that, though I might have just missed it. If we can modify the docker file so we can lock the Chrome version and then pin the Chromedriver version in `package.json`, then only upgrade them when both are released, it could work I think.